documentation/automating_windows_targets.md

```diff
@@ -168,269 +169,81 @@ documentation.](https://puppet.com/docs/pdk/1.x/pdk_overview.html)
-- Write a Bolt plan to apply Puppet code and orchestrate the deployment of a
-  package resource using the Chocolatey provider.
+This example installs the Frogsay package on a Windows target.
 
-### 1. Build a project-specific configuration
+**Before you begin:**
+- [Install Bolt](bolt_installing_modules.md)
+- Configure Windows Remote Management (WinRM) on your Windows target.
 
-Bolt runs in the context of a [Bolt project
-directory](projects.md). This directory contains all of the
-configuration, code, and data loaded by Bolt.
-
-1. Create a module called `bolt_choco_example`
-
-   ```
-   mkdir bolt_choco_example
-   ```
+To install a package with Chocolatey:
+1. Install Chocolatey on your Windows target using the `apply` command:
+   _\*nix shell command_
 
-1. Add a `bolt-project.yaml` file to the `puppet_choco_tap` directory:
-
-   ```
-   New-Item -Type File -Path .\puppet_choco_tap\bolt-project.yaml
-   ```
-
-   Adding a `bolt-project.yaml` file (even if it's empty),  makes the containing directory
-   a Bolt project directory when you run Bolt from it. This is where Bolt loads
-   code and configuration from.
-
-1. Create an inventory file to store information about your targets. This is
-   stored as `inventory.yaml` by default in the project directory. Add the
-   following code: 
-
-   ```
-   groups:
-     - name: windows
-       targets:
-         - chocowin0.classroom.puppet.com
-         - chocowin1.classroom.puppet.com
-       config:
-         transport: winrm
-         winrm:
-           user: Administrator
-           password: <ADD PASSWORD>
-   ```
-
-1. To make sure that your inventory is configured correctly and that you can
-   connect to all the targets, run the following command from inside the project
-   directory: 
-
-   ```
-   bolt command run 'echo hi' --targets windows
-   ```
-
-   **Note:** The `--targets windows` argument refers to the target group defined in
-   the inventory file.
-
-   You should get the following output:
-
-   ```
-   Started on x.x.x.x...
-   Started on localhost...
-   Finished on localhost:
-     STDOUT:
-       hi
-   Finished on 0.0.0.0:
-     STDOUT:
-       hi
-   Finished on 127.0.0.1:
-     STDOUT:
-       hi
-   Successful on 3 targets: 0.0.0.0:20022,localhost
-   Ran on 3 targets in 0.20 seconds
-   ```
-
-### 2. Download the Chocolatey module
-
-Bolt uses a [Puppetfile](https://puppet.com/docs/pe/latest/puppetfile.html) to
-install module content from the Forge. A `Puppetfile` is a formatted text file
-that specifies the modules and data you want in each environment.
-
-1. Create a file named `Puppetfile` in the project directory, with the modules
-   needed for this example:
-
-   ```     
-   mod 'puppetlabs-chocolatey', '4.1.0'
-   mod 'puppetlabs-stdlib', '4.13.1'
-   mod 'puppetlabs-powershell', '2.3.0'
-   mod 'puppetlabs-registry', '2.1.0'
+    ```shell
+    bolt apply -e 'include chocolatey' -t <TARGET URI> -u <USER> -p <PASSWORD> --transport winrm --no-ssl
```

Comment:
```suggestion
    bolt apply -e 'include chocolatey' -t <TARGET URI> -u <USER> -p <PASSWORD> --transport winrm
```
We've gotten complaints before about encouraging turning ssl off :P I don't particularly care either way, but we should at lest have the powershell command and nix one match

Reply:
Ah cool ty. Sidenote: I actually assumed it's off by default with Powershell because of the way -Ssl is described in the cmdlets reference. Is that true or should we look into rewording that?

Reply:
They should both default to `true` https://puppet.com/docs/bolt/latest/bolt_transports_reference.html#ssl